rust_src/remacs-sys/lib.rs

```diff
@@ -1115,10 +1115,28 @@ extern "C" {
     pub static Vminibuffer_list: Lisp_Object;
     pub static Vprocess_alist: Lisp_Object;
 
+    pub fn list1(arg1: Lisp_Object) -> Lisp_Object;
```

Comment:
These are all covered by the `list!` macro defined in eval_macros.rs. If you see a function/macro defined in C with a number in the name like `list3` or `call3` there is probably a macro in Rust for it or there should be.

Reply:
Macros are not explicitly imported so you won't see them as easily.

Reply:
I see. I never knew there was such a thing. I'll be sure to fix it.